runtime/src/commonMain/kotlin/pbandk/ExtendableMessage.kt

```diff
@@ -0,0 +1,21 @@
+package pbandk
+
+@Suppress("UNCHECKED_CAST")
+interface ExtendableMessage : Message {
+    val extensionFields: pbandk.AtomicReference<Map<Int, Any>>
```

Comment:
I think you forgot to add `@PbandkInternal` over here. You only added it in the generated code. And I wonder if it _is_ added here, is it still needed in the generated code?

Reply:
I did add it here but it doesn't do anything so I ripped it out. I'm happy to add it back but only the generated code utilizes it.

Reply:
What do you mean by "it doesn't do anything"? The use case I have in mind is code outside of pbandk that's operating on an `ExtendableMessage` (rather than a specific generated type). Without this annotation in place, will the compiler still disallow that code from accessing the `extensionFields` value of the `ExtendableMessage`? I suspect not.

Reply:
I see, I didn't think of that use case. That makes sense, it's added back.